zilencer/models.py

```diff
@@ -32,6 +32,7 @@ class RemotePushDeviceToken(AbstractPushDeviceToken):
     server: RemoteZulipServer = models.ForeignKey(RemoteZulipServer, on_delete=models.CASCADE)
     # The user id on the remote server for this device device this is
     user_id: int = models.BigIntegerField(db_index=True)
+    encrypt_notifications: bool = models.BooleanField(default=False)
```

Comment:
This name is confusing -- this field represents whether notifications ARE encrypted; the push bouncer shouldn't care.  We should add a comment explaining it.